scipy/signal/filter_design.py

```diff
@@ -248,6 +248,84 @@ def freqz(b, a=1, worN=None, whole=0, plot=None):
     return w, h
 
 
+def grpdelay(b, a=1, w=None, whole=False):
+    r"""Compute the group delay of a digital filter.
+
+    The group delay shows by how many samples different spectral components of
+    a signal are delayed by a filter. It is formally defined as
+    :math:`D(\omega) = -\frac{d}{d \omega} \arg H(e^{j \omega})`, where
+    :math:`H(e^{j \omega})` is the frequency response of a filter and
```

Comment:
We prefer to limit usage of LaTeX to the Notes section of the docstring and use plan text and/or a formula readable in a terminal here. Could you move it there?